### PR TITLE
Do not throw an exception on missing `mrack.log`

### PR DESCRIPTION
**Pull request by Tiboris**
os.stat throws exception so we use try except when removing file to accept the fact when the file does not exists

### CODE REVIEW

File: tmt/steps/provision/mrack.py [hunk 1]
```diff
@@ -91,8 +92,13 @@ def import_and_load_mrack_deps(workdir: Any, name: str) -> None:
         # HAX remove mrack stdout and move the logfile to /tmp
         mrack.logger.removeHandler(mrack.console_handler)
         mrack.logger.removeHandler(mrack.file_handler)
-        if os.stat("mrack.log"):
+
+        try:
             os.remove("mrack.log")
+        except OSError as e:
+            if e.errno != errno.ENOENT:  # errno.ENOENT = no such file or directory
+                ProvisionError(str(e))
```
Comment (happz) [inline tmt/steps/provision/mrack.py:100]: A more descriptive message would be better, to provide higher level of context to the user, plus `… from e` to preserve the original exception.

Comment (happz) [reply tmt/steps/provision/mrack.py]: Warning sounds also good. If it's not critical, it's not necessary to kill the whole run at the finish line because of a missing log.

Comment (Tiboris) [reply tmt/steps/provision/mrack.py]: I hope this version is more satisfying. @dav-pascual please have a look as well.

Comment (lukaszachy) [reply tmt/steps/provision/mrack.py]: pre-commit complains about unused import however except that it looks good.

Comment (happz) [reply tmt/steps/provision/mrack.py]: Shouldn't there supposed to be a warning? In the most recent version, I see an explicit `pass`, effectively silencing the exception. Is that correct?

Comment (psss) [reply tmt/steps/provision/mrack.py]: If I understand @Tiboris correctly, this is just a conditional cleanup of the log file which just might not be present (and it's not a problem). So I'd say don't want to bother user with any warning, right? Or did I get it wrong?

**Comment by psss** [comment]
@Tiboris, could you please remove the prefix from the commit message? We're not using them to have a consistent release notes. See also the [commit](https://tmt.readthedocs.io/en/latest/contribute.html#commits) recommendations. Thanks.

File: tmt/steps/provision/mrack.py (hunk 1 above)
Comment (Tiboris) [reply tmt/steps/provision/mrack.py]: Yes correct let me rephrase commit

**Comment by lukaszachy** [comment]
@happz @Tiboris I've fixed according to what ruff complained. Please ack/nack.

**Review by happz — APPROVED** [review]

**Review by dav-pascual — APPROVED** [review]
I can ACK on behalf of @Tiboris

**Comment by lukaszachy** [comment]
> I can ACK on behalf of @Tiboris

Thank you.

**Review by lukaszachy — APPROVED** [review]
